site/tests/test_components.py

```diff
@@ -77,3 +84,243 @@ def test_subjects_validation_component_update_draft_admin(
         component.update_draft(administrator.identity, data=new_data, record=draft)
         == None
     )
+
+
+def test_mint_alternate_identifier_component(
+    minimal_record_with_files, uploader, client, administrator, monkeypatch
+):
+    """Test for the mint alternative identifier component.
+
+    Tests all scenarios:
+    1. Normal case - single identifier
+    2. Update draft validation
+    3. Same scheme, different values
+    4. Same value, different schemes
+    5. Mixed mintable and non-mintable schemes
```

Comment:
to add: mixed mintable and non mintable schemes with a validation error on the mintable - will the non-mintable values be preserved on the draft?